app.py

```diff
@@ -0,0 +1,76 @@
+""" This script loads in a model, gets a Url, gets an article in the URL and summarizes the article """
+
+import pickle as p
+import traceback
+import json
+from flask import Flask, request, jsonify
+from flask_cors import CORS
+
+# loading the model
+summarizer = 'model.pkl'
+model = p.load(open(summarizer, 'rb'))
+
+app = Flask(__name__)
+CORS(app)
+
+
+@app.route('/api/summarize', methods=['POST', 'GET'])
+def summarize():
+    """ Returns summary of articles """
+    if model:
+        try:
+            if request.method == 'POST':
+                url = request.json['theUrl']

```

Comment:
The task 9 function can start here with url = request.json['theUrl'] as the first line.